benchmark

```diff
@@ -34,3 +32,4 @@ cd webdriver-ts
 npm run selenium -- --framework vanillajs-keyed reflex --count 1 --headless --chromeBinary "$(which chromium)" --chromeDriver "$(which chromedriver)"
 npm run results
 kill $SERVER_PID
+rm -rf $BENCHMARK
```

Comment:
https://github.com/reflex-frp/reflex-platform/blob/c6c60935b52d68320358f9736e3fd2207943d86e/run-in-commit#L11-L12